UI/web-src/ngMango/piMango.css

```diff
@@ -0,0 +1,165 @@
+    background-color: var(--ma-primary);
+    color: var(--ma-primary-contrast);
+}
+
+.md-button.md-pimangoDark-theme.md-hue-1.md-accent.md-fab:not([disabled]).md-focused md-icon, .md-button.md-pimangoDark-theme.md-hue-1.md-accent.md-fab:not([disabled]):hover md-icon, .md-button.md-pimangoDark-theme.md-hue-1.md-accent.md-raised:not([disabled]).md-focused md-icon, .md-button.md-pimangoDark-theme.md-hue-1.md-accent.md-raised:not([disabled]):hover md-icon {
+    color: var(--ma-primary-contrast);
+}
+
+.md-button.md-pimangoDark-theme.md-hue-1.md-primary.md-fab:not([disabled]).md-focused, .md-button.md-pimangoDark-theme.md-hue-1.md-primary.md-fab:not([disabled]):hover, .md-button.md-pimangoDark-theme.md-hue-1.md-primary.md-raised:not([disabled]).md-focused, .md-button.md-pimangoDark-theme.md-hue-1.md-primary.md-raised:not([disabled]):hover {
+    background-color: var(--ma-primary);
+    color: var(--ma-primary-contrast);
+}
+
+.md-button.md-pimangoDark-theme.md-hue-2.md-primary.md-fab:not([disabled]).md-focused, .md-button.md-pimangoDark-theme.md-hue-2.md-primary.md-fab:not([disabled]):hover, .md-button.md-pimangoDark-theme.md-hue-2.md-primary.md-raised:not([disabled]).md-focused, .md-button.md-pimangoDark-theme.md-hue-2.md-primary.md-raised:not([disabled]):hover {
+    background-color: var(--ma-accent);
+}
+
+md-toolbar.md-pimangoDark-theme.md-hue-2:not(.md-menu-toolbar) ma-drop-down-button:not([disabled]) > div  {
+    color: var(--ma-primary-contrast);
+}
+
+md-select-menu.md-pimangoDark-theme md-content md-option[selected].md-focused, md-select-menu.md-pimangoDark-theme md-content md-option[selected]:focus {
+    color: var(--ma-foreground-1);
+}
+
+md-tabs.md-pimangoDark-theme .md-tab.md-active, md-tabs.md-pimangoDark-theme .md-tab.md-active md-icon, md-tabs.md-pimangoDark-theme .md-tab.md-focused, md-tabs.md-pimangoDark-theme .md-tab.md-focused md-icon {
+    color: var(--ma-primary-contrast)
+}
+
+md-sidenav.site-sidenav md-content {
+    background-color: var(--ma-primary) !important;
+    color: var(--ma-primary-contrast) !important;
+}
+
+.ma-ui-menu {
+    background-color: var(--ma-primary) !important;
+    color: var(--ma-primary-contrast) !important;
+}
+
+md-toolbar.md-pimangolight-theme:not(.md-menu-toolbar) {
+    background-color: var(--ma-background) !important;
+    color: var(--ma-background-contrast) !important;
+}
+
+md-toolbar.md-tall.ma-ui-sidebar-header {
+    background-color: var(--ma-primary) !important;
+    color: var(--ma-primary-contrast) !important;
+}
+
+md-content.md-pimangolight-theme {
+    background-color: var(--ma-background-hue-3) !important;
+    color: var(--ma-background-hue-3-contrast) !important;
+}
+
+md-toolbar.md-pimangoDark-theme.md-hue-3:not(.md-menu-toolbar) {
+    background-color: var(--ma-background-hue-1) !important;
+    color: var(--ma-background-hue-1-contrast) !important;
+}
```

Comment:
Are all the `!important` necessary? Its generally best to avoid this unless absolutely necessary. I assume you are trying to override a rule in the default Angular Material CSS? Make sure this rule has the same level of specificity as the default and it should override it.

Reply:
I removed all the importants